testdata/breakOnLoad_firstLineBp/index.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
```

Comment:
Do you need to add new testdata scripts for this? Use an existing script if possible. Otherwise name this for the general scenario that the test script covers, not for the specific test that it was added for.

Reply:
Merged with an existing scenario